### PR TITLE
readme edit - words, links,  and port spec fixes

### PR DESCRIPTION
**Pull request by footloosejava**
- Reworded a bit
- Changed ports in chart to reflect typical setup conditions.
- Added API getting started link
- Added a little more detail in command line
- Added short explainer about friending within the network

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,13 +50,23 @@ $ mvn clean compile
...
 
 #### Locally
 
-Running IRI is pretty simple, and you don't even have to run it under admin rights. Below is a list of command line options. Here is an example script:
+Running IRI is quick and easy, and you can usually run it without admin rights.
+Below is a list of command line options.
+At a minimum you must provide one of:
+
+- the port — e.g., '`-p 14265`'
+- the location of an ini config file — e.g., '`-c iota.ini`'
```
Comment (GalRogozinski) [inline README.md]: Delete this line please. The `-c` option doesn't really work. The way we will give an external config is via a known name and location.

File: README.md [hunk 2]
```diff
@@ -33,13 +50,23 @@ $ mvn clean compile
...
 #### Locally
 
-Running IRI is pretty simple, and you don't even have to run it under admin rights. Below is a list of command line options. Here is an example script:
+Running IRI is quick and easy, and you can usually run it without admin rights.
+Below is a list of command line options.
+At a minimum you must provide one of:
+
+- the port — e.g., '`-p 14265`'
+- the location of an ini config file — e.g., '`-c iota.ini`' 
+- the default config ini file must exist **and also** contain the port setting — e.g.,'`PORT = 14265`'.
```
Comment (GalRogozinski) [inline README.md]: I think "must" is a strong word here. I can run IRI without it. Also, if PORT is missing from the ini then IRI can run if it is given via CLI.

File: README.md [hunk 3]
```diff
@@ -57,29 +84,28 @@ command line arguments.
 
 Option | Shortened version | Description | Example Input
 --- | --- | --- | --- 
-`--port` | `-p` | This is a *mandatory* option that defines the port to be used to send API commands to your node | `-p 14800`
+`--port` | `-p` | This is a *mandatory* option that defines the port to be used to send API commands to your node | `-p 14265`
 `--neighbors` | `-n` | Neighbors that you are connected with will be added via this option. | `-n "udp://148.148.148.148:14265 udp://[2001:db8:a0b:12f0::1]:14265"`
```
[the remark this answers is not in the document]
Comment (GalRogozinski) [reply README.md]: :+1:

File: README.md [hunk 4]
```diff
@@ -57,29 +84,28 @@ command line arguments.
...
 ```
 [IRI]
-PORT = 14700
-UDP_RECEIVER_PORT = 14700
-NEIGHBORS = udp://my.favorite.com:15600
+PORT = 14265
+UDP_RECEIVER_PORT = 14600
+NEIGHBORS = udp://my.favorite.com:14600
 IXI_DIR = ixi
 HEADLESS = true
```
Comment (GalRogozinski) [inline README.md]: I am pretty sure the HEADLESS config is non existent now.
@alon-e, you know what it was for?

anyhows, you can remove it.

File: README.md [hunk 5]
```diff
@@ -57,29 +84,28 @@ command line arguments.
...
-DEBUG = true
+DEBUG = false
```
Comment (GalRogozinski) [inline README.md]: :+1:

**Comment by footloosejava** [comment]
@GalRogozinski OK - see now.

**Comment by footloosejava** [comment]
See new PR. My git went berserk!

**Pull request closed without merging**